vllm_ascend/envs.py

```diff
@@ -34,6 +34,10 @@
     lambda: os.getenv("CMAKE_BUILD_TYPE"),
     "COMPILE_CUSTOM_KERNELS":
     lambda: bool(int(os.getenv("COMPILE_CUSTOM_KERNELS", "1"))),
+    "VLLM_ENABLE_MC2":
+    lambda: bool(int(os.getenv("VLLM_ENABLE_MC2", '0'))),
```

Comment:
There is a pr to fix MC2 problem already. Please check if it works for your case. https://github.com/vllm-project/vllm-ascend/pull/710